test/fixtures/3.0/ping/README.md

```diff
@@ -0,0 +1,3 @@
+# ping
```

Comment:
Please rework the content of this README to explain what is this directory providing in the context of loopback-connector-swagger.